// @tensorflow-models/body-pix/dist/index.js
"use strict";
/**
 * @license
 * Copyright 2019 Google Inc. All Rights Reserved.
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * https://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 * =============================================================================
 */
Object.defineProperty(exports, "__esModule", { value: true });
var body_pix_model_1 = require("./body_pix_model");
exports.BodyPix = body_pix_model_1.BodyPix;
exports.load = body_pix_model_1.load;
var output_rendering_util_1 = require("./output_rendering_util");
exports.blurBodyPart = output_rendering_util_1.blurBodyPart;
exports.drawBokehEffect = output_rendering_util_1.drawBokehEffect;
exports.drawMask = output_rendering_util_1.drawMask;
exports.drawPixelatedMask = output_rendering_util_1.drawPixelatedMask;
exports.toColoredPartMask = output_rendering_util_1.toColoredPartMask;
exports.toMask = output_rendering_util_1.toMask;
var part_channels_1 = require("./part_channels");
exports.PART_CHANNELS = part_channels_1.PART_CHANNELS;
var util_1 = require("./util");
exports.flipPoseHorizontal = util_1.flipPoseHorizontal;
exports.resizeAndPadTo = util_1.resizeAndPadTo;
exports.scaleAndCropToInputTensorShape = util_1.scaleAndCropToInputTensorShape;
var version_1 = require("./version");
exports.version = version_1.version;
